docs/api/search-query-service-resource.md

```diff
@@ -143,9 +143,11 @@ name      | string  | yes      | The name of the package type.
 ### Sample request
 
 ```
-GET https://azuresearch-usnc.nuget.org/query?q=NuGet.Versioning&prerelease=false&semVerLevel=2.0.0
+GET https://search-sample.nuget.org/query?q=NuGet.Versioning&prerelease=false&semVerLevel=2.0.0
```

Comment:
Same as https://github.com/NuGet/docs.microsoft.com-nuget/pull/2971/files#r1054651582.
How to get this value if someone coming to this page 1st time?

Reply:
https://github.com/NuGet/docs.microsoft.com-nuget/pull/2971#discussion_r1054656517